From 09a9a4fdec9ee709bd3da2be2ba1cf13e4e377e1 Mon Sep 17 00:00:00 2001
From: Harald Judt <h.judt@gmx.at>
Date: Wed, 20 Feb 2013 15:25:09 +0100
Subject: [PATCH] Fix color parsing when reading config file.

This turned out to be a stupid copy & paste mistake.

(cherry picked from commit 462e85dfd54e9a3bdf6c330c602a6545bccce585)
---
 panel-plugin/weather.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panel-plugin/weather.c b/panel-plugin/weather.c
index 710c33c..4a2e3c3 100644
--- a/panel-plugin/weather.c
+++ b/panel-plugin/weather.c
@@ -884,7 +884,7 @@ xfceweather_read_config(XfcePanelPlugin *plugin,
 
     value = xfce_rc_read_entry(rc, "scrollbox_color", NULL);
     if (value)
-        gdk_color_parse("#rrrrggggbbbb", &(data->scrollbox_color));
+        gdk_color_parse(value, &(data->scrollbox_color));
 
     data->scrollbox_use_color =
         xfce_rc_read_bool_entry(rc, "scrollbox_use_color", FALSE);
-- 
2.1.2

